apps/web/composables/useModernImage/__tests__/useModernImage.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { mockNuxtImport } from "@nuxt/test-utils/runtime";
+import { useModernImage } from "../useModernImage";
+
+const { useRuntimeConfig } = vi.hoisted(() => {
+    return {
+        useRuntimeConfig: vi.fn().mockImplementation(() => {
+            return {
+                public: {
+                    useAvif: true,
+                    useWebp: true
+                }
+            }
+        })
```

Comment:
**suggestion**: You don't need this part. I would remove it because you define the state of the mock below.

Reply:
done, was only able to remove public: {...}